PizzaBulider/Program.cs

```diff
@@ -0,0 +1,169 @@
+            return result;
+        }
+    }
+    public class WheatPizzaBuilder : IPizzaBuilder
+    {
+        private WheatPizza WheatPizza = new WheatPizza();
+
+        public WheatPizzaBuilder()
+        {
+            this.Reset();
+        }
+        public void Reset()
+        {
+            this.WheatPizza = new WheatPizza();
+        }
+        public void BuildDough()
+        {
+            this.WheatPizza.AddIngredient("Dough");
+        }
+        public void BuildSauce()
+        {
+            this.WheatPizza.AddIngredient("Sauce");
+        }
+        public void BuildTopping()
+        {
+            this.WheatPizza.AddIngredient("Topping");
+        }
+        public WheatPizza GetPizza()
+        {
+            WheatPizza final = this.WheatPizza;
+            this.Reset();
+            return final;
+        }
+    }
+    public class WhitePizze
+    {
+        private List<object> Ingredients = new List<object>();
+        public void AddIngredient(object Ingredient)
+        {
+            Ingredients.Add(Ingredient);
+        }
+        public string ListIngredients()
+        {
+            string str = "";
+            for (int i = 0; i < this.Ingredients.Count; i++)
+            {
+                str += this.Ingredients[i] + "\n";
+            }
+            return "White Pizza order : \n" + str;
+        }
+    }
+    public class WheatPizza
+    {
+        private List<object> Ingredients = new List<object>();
+        public void AddIngredient(object Ingredient)
+        {
+            Ingredients.Add(Ingredient);
+        }
+        public string ListIngredients()
+        {
+            string str = "";
+            for (int i = 0; i < this.Ingredients.Count; i++)
+            {
+                str += this.Ingredients[i] + "\n";
+            }
+            return "Wheat Pizza order : \n" + str;
+        }
+
+    }
+
+    public class Director
+    {
+        private IPizzaBuilder _pizzaBuilder;
+        public IPizzaBuilder PizzaBuilder
+        {
+            set { this._pizzaBuilder = value; }
+        }
+        public void makePizza()
+        {
+            this._pizzaBuilder.BuildDough();
+            this._pizzaBuilder.BuildSauce();
+            this._pizzaBuilder.BuildTopping();
+        }
+        public void makePizzaWithoutToppings()
+        {
+            this._pizzaBuilder.BuildDough();
+            this._pizzaBuilder.BuildSauce();
+        }
+    }
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Welcome to Norah pizza store :) ");
+            Console.Write("How would you like to run the program? Enter 0 for the main objectives or 1 for Bonus 1: ");
+            var UserInput  = Console.ReadLine();
+                switch (UserInput)
+                {
+                    case "0":
+                        Director director = new Director();
+                        WheatPizzaBuilder Wheat = new WheatPizzaBuilder();
```

Comment:
It would be better if you used polymorphism to define builders, without creating new builder variable for each one.